### ISSUE TITLE
Play a jingle at a fixed time

### ISSUE THREAD

**Issue by LordHelmchen666** [AzuraCast/AzuraCast#2905]
Hi, i found this article in development version docs of Liquidsoap. https://www.liquidsoap.info/doc-dev/cookbook.html

> **Play a jingle at a fixed time**
> 
> Suppose that we have a playlist jingles of jingles and we want to play one within the 5 first minutes of every hour, without interrupting the current song. We can think of doing something like
> 
> `radio = switch([({ 0m-5m }, jingles), ({ true }, playlist)])`
> 
> but the problem is that it is likely to play many jingles. In order to play exactly one jingle, we can use the function **_predicate.activates_** which detects when a predicate (here { 0m-5m }) becomes true:
> 
> `radio = switch([(predicate.activates({ 0m-5m }), jingles), ({ true }, playlist)])`

Is this code working with Liquidsoap version shipped with Azuracast?

**Comment by ungaro** [comment]
well, i was using this and it was working until somehow it decided not to work:
`
jingles_over = audio_to_stereo(playlist("/var/azuracast/stations/<station_name>/playlists/playlist_jingles.m3u"))

radio  = smooth_add(
        normal= radio,
        p=0.70,
        special=switch([
        ( { 0m0s }, jingles_over )
        ])
)
`

Added via Utilities->Edit Liquidsoap Configuration. (scroll down to bottom and paste into last textinput, create a playlist named jingles and add your jingle there) If someone can tell me why it doesn't work anymore, I will be glad.

i have triple checked that there is a file inside playlists folder. Maybe you can try this? @LordHelmchen666

**Comment by d-trax** [comment]
Hi, I am trying to make my radio station to play jingles without interrupting the music (on top of the music) and I came across your post. Tried your scripts, but none of them worked. Found some more similar scripts on the web, but none of them worked as well, giving me similar errors (<img width="691" alt="Screen Shot 2021-08-17 at 6 51 18 PM" src="https://user-images.githubusercontent.com/31935215/129759898-d944a590-5032-4759-bada-f8e1400e2e8f.png"> a screenshot). Did you guys managed to make it work? 
Tried with manually create a scheduled playlist for the jingles with a higher weight and 'Once per minute' option, but nothing worked. 
P.S. I am very new with AzuraCast and scripting, so any help will be highly appreciated. xD
